Makefile

```
@@ -282,6 +282,12 @@ validate-test-html:
 	@$(DEVSHELL) html5validator tests/functional/pageslayout/html
 	@echo
 
+.PHONY: accessibility-summary
+accessibility-summary:
+	@echo "███ Processing accessibility results..."
+	@$(DEVSHELL) $(SDBIN)/summarize-accessibility-info
+	less securedrop/tests/functional/pageslayout/accessibility-info/summary.txt
```

I suggest leaving this up to the user via `make accessibility-summary | less`, since otherwise this path is opaque:

```suggestion
	cat securedrop/tests/functional/pageslayout/accessibility-info/summary.txt
```